### DISCUSSION TITLE
How can delete all the contents of a folder directory

### DISCUSSION THREAD

**Discussion by harrypo03** [dani-garcia/vaultwarden#4095] [Q&A]
How can I delete all the contents of a folder directory at once? I have not been able to find a way, please help me

**Comment by BlackDex** [comment]
Well, you can first select that folder, that should only show the ciphers in that folder, and then multi select them.

But there isn't a delete this folder and all containing ciphers.

In theory, folders are more like labels here.

**Comment by harrypo03** [comment]
&nbsp; &nbsp; Thank you for your reply and guidance, the situation is like this, I imported 1pass passwords, the labels in them are converted to sub-files under the folder (which seems to work for me), each sub-file has a certain number of passwords saved accordingly.
&nbsp; &nbsp; But I imported it three times (one wrong operation) so that there are three renamed subfiles in each of the folders with the same number of passwords in each subfile, so I would like to be able to delete them in bulk or delete them all at once and reimport them~
&nbsp; I don't know if there is a way to batch delete these subfiles other than deleting the account and resetting it, otherwise deleting them one by one would be a disaster~~~
&nbsp;Have a nice life and work ~ thanks again





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dani-garcia/vaultwarden"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月23日(星期四) 下午3:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dani-garcia/vaultwarden] How can delete all the contents of a folder directory (Discussion #4095)





 
Well, you can first select that folder, that should only show the ciphers in that folder, and then multi select them.
 
But there isn't a delete this folder and all containing ciphers.
 
In theory, folders are more like labels here.

**Comment by kozmotronik** [comment]
Same here. I have imported 2 times, each entry duplicated. I've deleted all entries in bulk but their folders left. So I am deleting them one by one. To be able to delete in bulk would be a life saver.
